scale/product/migrations/0008_auto_20170221_1413.py

```diff
@@ -12,15 +12,15 @@ class Migration(migrations.Migration):
 
     def populate_file_ancestry_link_batch(apps, schema_editor):
         # Go through all of the BatchRecipe models and update the corresponding FileAncestryLink models
-        BatchRecipe = apps.get_model('batch', 'BatchRecipe')
+        Recipe = apps.get_model('recipe', 'Recipe')
```

Comment:
Is this function necessary now that BatchRecipe has been removed? I'm assuming this is to not lose any file ancestry that was once stored in the BatchRecipe?

Reply:
You're right, but do we delete the whole migration then?  We'd have to update the migrations that follow it and make sure they don't get reapplied.  Maybe it's simpler than I realize.

Reply:
Tossed it in the fire